src/transformers/models/luke/tokenization_luke.py

```diff
@@ -0,0 +1,1413 @@
+    r"""
+    Construct a LUKE tokenizer.
+
+    This tokenizer inherits from :class:`~transformers.RobertaTokenizer` which contains most of the main methods. Users
+    should refer to this superclass for more information regarding those methods. Compared to
+    :class:`~transformers.RobertaTokenizer`, :class:`~transformers.LukeTokenizer` also creates entity sequences, namely
+    :obj:`entity_ids`, :obj:`entity_attention_mask`, :obj:`entity_token_type_ids` and :obj:`entity_position_ids` to be
+    used by the LUKE model.
+
+    Args:
+        vocab_file (:obj:`str`):
+            Path to the vocabulary file.
+        merges_file (:obj:`str`):
+            Path to the merges file.
+        entity_vocab_file (:obj:`str`):
+            Path to the entity vocabulary file.
+        task (:obj:`str`, `optional`):
+            Task for which you want to prepare sequences. One of "entity_classification" or
+            "entity_pair_classification". If you specify this argument, the entity sequence is automatically created
+            based on the given entity spans.
+        max_entity_length (:obj:`int`, `optional`, defaults to 32):
+            The maximum length of :obj:`entity_ids`.
+        max_mention_length (:obj:`int`, `optional`, defaults to 30):
+            The maximum number of tokens inside an entity span.
+        entity_token_1 (:obj:`str`, `optional`, defaults to :obj:`<ent>`):
+            The special token representing an entity span. This token is only used when `task` is set to
+            "entity_classification" or "entity_pair_classification".
+        entity_token_2 (:obj:`str`, `optional`, defaults to :obj:`<ent2>`):
+            The special token representing an entity span. This token is only used when `task` is set to
+            "entity_pair_classification".
+    """
+
+    vocab_files_names = VOCAB_FILES_NAMES
+    pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
+    max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
+
+    def __init__(
+        self,
+        vocab_file,
+        merges_file,
+        entity_vocab_file,
+        task=None,
+        max_entity_length=32,
+        max_mention_length=30,
+        entity_token_1="<ent>",
+        entity_token_2="<ent2>",
+        **kwargs
+    ):
+        # we add 2 special tokens for downstream tasks
+        # for more information about lstrip and rstrip, see https://github.com/huggingface/transformers/pull/2778
+        entity_token_1 = (
+            AddedToken(entity_token_1, lstrip=False, rstrip=False)
+            if isinstance(entity_token_1, str)
+            else entity_token_1
+        )
+        entity_token_2 = (
+            AddedToken(entity_token_2, lstrip=False, rstrip=False)
+            if isinstance(entity_token_2, str)
+            else entity_token_2
+        )
+        kwargs["additional_special_tokens"] = [entity_token_1, entity_token_2] + kwargs.get(
+            "additional_special_tokens", []
+        )
```

Comment:
hard to read this, can we make this two lines instead ?

Reply:
This comment should be addressed, I agree. we can have 
```
kwargs["additional_special_tokens"] = [entity_token_1, entity_token_2]
kwargs["additional_special_tokens"] += kwargs.get("additional_special_tokens", [])
```

Reply:
Ok, done